NitroxClient/GameLogic/Entities.cs

```diff
@@ -66,9 +66,16 @@ public void Spawn(List<Entity> entities)
                 }
                 else
                 {
-                    Optional<GameObject> parent = NitroxEntity.GetObjectFrom(entity.ParentId);
-                    Spawn(entity, parent);
-                    SpawnAnyPendingChildren(entity);
+                    try
+                    {
+                        Optional<GameObject> parent = NitroxEntity.GetObjectFrom(entity.ParentId);
+                        Spawn(entity, parent);
+                        SpawnAnyPendingChildren(entity);
+                    }
+                    catch(OptionalEmptyException<GameObject> e)
+                    {
+                        Log.Error($"Failed to spawn Entity {entity.Id}, a {entity.TechType}: {e.Message}");
+                    }
```

Comment:
?

Reply:
Unrelated change. Problem here was that when spawning of a single entity failed, the whole entity spawner failed and would end up as stalling login.